.github/workflows/ci-backend.yml

```diff
@@ -155,84 +197,58 @@ jobs:
-                  touch frontend/dist/shared_dashboard.html
-                  pytest ee -m "not saml_only" --cov --cov-report=xml:coverage-clickhouse.xml
                   pytest posthog -m "ee"
+
             - uses: codecov/codecov-action@v2
+              if: ${{ !matrix.saml && !matrix.cloud }}
               with:
                   files: ./coverage-postgres.xml,./coverage-clickhouse.xml
                   fail_ci_if_error: false
                   verbose: true
 
-    saml:
-        needs: changes
-        if: ${{ needs.changes.outputs.backend == 'true' && github.repository == 'PostHog/posthog' }}
-
-        name: Django tests – with SAML
-        runs-on: ubuntu-latest
-        steps:
-            - uses: actions/checkout@v1
-              with:
-                  fetch-depth: 1
-
-            - name: Start stack with Docker Compose
-              run: docker-compose -f ee/docker-compose.ch.yml up -d db clickhouse zookeeper kafka
-
-            - name: Set up Python
-              uses: actions/setup-python@v2
+            - name: Upload updated timing data as artifacts
+              uses: actions/upload-artifact@v2
+              if: ${{ matrix.ee }}
               with:
-                  python-version: 3.9
-
-            - name: Install SAML (python3-saml) dependencies (not required for Cloud or FOSS)
-              run: sudo apt-get install libxml2-dev libxmlsec1-dev libxmlsec1-openssl
-
-            - uses: syphar/restore-virtualenv@v1.2
-              id: cache-backend-tests-saml
-
-            - uses: syphar/restore-pip-download-cache@v1
-              if: steps.cache-backend-tests-saml.outputs.cache-hit != 'true'
-
-            - name: Install python dependencies
-              if: steps.cache-backend-tests-saml.outputs.cache-hit != 'true'
-              run: |
-                  python -m pip install -r requirements-dev.txt
-                  python -m pip install -r requirements.txt
-                  python -m pip install python3-saml==1.12.0
-
-            - name: Add kafka host to /etc/hosts for kafka connectivity
-              run: sudo echo "127.0.0.1 kafka" | sudo tee -a /etc/hosts
-
-            - name: Run posthog tests
-              run: |
-                  mkdir -p frontend/dist
-                  touch frontend/dist/index.html
-                  touch frontend/dist/layout.html
-                  touch frontend/dist/shared_dashboard.html
-                  pytest -m "not ee" posthog/
-            - name: Run EE tests
-              env:
-                  PRIMARY_DB: 'clickhouse'
-              run: |
-                  mkdir -p frontend/dist
-                  touch frontend/dist/index.html
-                  touch frontend/dist/layout.html
-                  touch frontend/dist/shared_dashboard.html
-                  pytest ee/
-                  pytest posthog -m "ee"
+                  name: timing_data-${{ matrix.group }}
+                  path: .test_durations
```

Comment:
Looks like we're saving this as an artifact but the file is also committed to the repo, do we want to keep it there? Will be outdated pretty quickly.

Reply:
See PR description - this is a follow-up item. TBH it won't be that out of date that quick.